### DISCUSSION TITLE
Template variables advanced date customization

### DISCUSSION THREAD

**Discussion by NoodleNodes** [notaapp/talk#278] [Questions] [answered]
Hi @astoilkov , 

Just need a little nudge in the right direction please. I'm trying to setup a daily note template with links to the previous and next day daily notes. 

`[[ 2022-08-02 Tuesday]] | **2022-08-03 Wednesday** |  [[2022-08-04 Thursday]]`

In Obsidian it looked like this: 

`[[ <% tp.date.now("YYYY-MM-DD dddd -1", -1, tp.file.title, "YYYY-MM-DD") %>]] | **<% tp.file.title %>** |  [[<% tp.date.now("YYYY-MM-DD dddd", 1, tp.file.title, "YYYY-MM-DD") %>]]

Is this possible to replicate in Nota with the Day.js syntax you provide in your documentation?

In addition, is it possible to incorporate this `<% tp.web.daily_quote() %>` which pulls a daily quote from the web?

```
> Only through our connectedness to others can we really know and enhance the self. And only through working on the self can we begin to enhance our connectedness to others.
> — <cite>Harriet Lerner</cite>
```

Thanks for the help in advance 😊🙏

**Comment by astoilkov** [comment, marked as answer]
That's not currently possible. In Obsidian, you need to use the Templater plugin for that. However, we can't build the same functionality as Templater because there would be security issues:
<img width="711" alt="CleanShot 2022-08-04 at 10 00 56@2x" src="https://user-images.githubusercontent.com/884810/182784164-e9c8596b-bb39-49c3-bb39-d97c4ef5527a.png">

We are exploring ways that would allow you to do this but without the security concerns.

**Reply by NoodleNodes** [reply]
Thank you for getting back to me on this. No problem, I'd rather maintain a secure environment 👍 I'll see if I can use Keyboard Maestro to automate this. Just wanted to check if there was a way to manipulate the date within the template before going the route of a third party app 🤪

**Reply by astoilkov** [reply]
We are considering this: https://github.com/notaapp/talk/issues/201. It will allow you to do what you want.

**Reply by NoodleNodes** [reply]
> We are considering this: https://github.com/notaapp/talk/issues/201. It will allow you to do what you want.

Thank you for the link, I just had a look. This would work, part of the way, but I'm wanting to either + or - a day from the {{date}} variable. The website you refer to in your documentation (Day.js) has a section on manipulating the date variable so I assumed it might work thinking Nota was pulling from Day.js in the background somehow not just to be used as a formatting reference guide. I don't know enough to give an informed opinion 😜 this kind of thing leaves me scratching my head and looking for a safe place to assume the fetal position 😂
